ide/src/focus.ts

```diff
@@ -26,8 +28,9 @@ interface FocusState {
 }
 
 let state: FocusState | null = null;
+let doc_save_callback: vscode.Disposable;
 
-let initialize = async (call_flowistry: CallFlowistry) => {
+let initialize = async (call_flowistry: CallFlowistry, hide_error?: boolean) => {
```

Comment:
The `hide_error` flag prevents the compiler output from displaying each time the user saves the document while in focus mode.